website/static/js/contribManager.js

```diff
@@ -462,11 +423,39 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
 
 function ContribManager(selector, contributors, adminContributors, user, isRegistration, table, adminTable) {
     var self = this;
+    //shouter allows communication between ContribManager and ContribRemover, in particular which contributor needs to
+    // be removed is passed to ContribRemover
+    var contribShouter = new ko.subscribable();
```

Comment:
If I try and use self.contribToRemove as a variable in the mako template, knockout does not work and you see that, all the buttons and no data populated.